server/index.js

```diff
@@ -47,7 +47,7 @@ app.use(loggingMiddleware);
 app.use(authMiddleware);
 
 // Sync Sequelize models
-db.sequelize.sync().then(async () => {
+db.sequelize.sync({ force: true }).then(async () => {
```

Comment:
for god's sake don't commit this. We will manually reset the db on the VM. If you commit this code, the database will drop on every rerun, that is every future commit. Meaning that we will loose all data every time we push a change to main.